src/plugins/new-ui/public/webcomponents/compass.html

```diff
@@ -48,143 +84,265 @@
+              <text id="text4965" fill$="{{ options.bezel }}" style="text-anchor:middle;text-align:center;" transform="matrix(1,0,0,1,460,-38)" line-height="125%" y="530" x="460">{{__('+90')}}</text>
+              <text id="text4967" fill$="{{ options.bezel }}" style="text-anchor:middle;text-align:center;" transform="matrix(1,0,0,1,-425,-35)" line-height="125%" y="530" x="460">{{__('-90')}}</text>
+              <text id="text4969" fill$="{{ options.bezel }}" style="text-anchor:middle;text-align:center;" transform="matrix(1,0,0,1,-320,300)" line-height="125%" font-size="72px" y="530" x="460">{{__('-135')}}</text>
+              <text id="text4971" fill$="{{ options.bezel }}" style="text-anchor:middle;text-align:center;" transform="matrix(1,0,0,1,-325,-350)" line-height="125%" font-size="72px" y="530" x="460">{{__('-45')}}</text>
+              <text id="text4973" fill$="{{ options.bezel }}" style="text-anchor:middle;text-align:center;" transform="matrix(1,0,0,1,325,-350)" line-height="125%" font-size="72px" y="530" x="460">{{__('+45')}}</text>
+              <text id="text4975" fill$="{{ options.bezel }}" style="text-anchor:middle;text-align:center;" transform="matrix(1,0,0,1,320,300)" line-height="125%" font-size="72px" y="530" x="460">{{__('+135')}}</text>
+            </g>
           </g>
 
           <text id="lock" fill$="{{ options.bezel }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="120px" letter-spacing="0" line-height="100%" word-spacing="0" y="280" x="460.67117" title="Heading hold target">{{__('LK')}}</text>
 
           <text id="heading" fill$="{{ options.headingFill }}" stroke$="{{ options.bezel }}" stroke-width$="{{ options.headingStrokeWidth }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="250px" letter-spacing="0"
           line-height="125%" word-spacing="0" y="530" x="460.67117" font-weight="bold">{{heading}}</text>
-
+          
           <text id="target" fill$="{{ options.bezel }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="150px" letter-spacing="0" line-height="100%" word-spacing="0" y="680" x="460.67117" font-weight="bold"
           title="Heading hold target">{{target}}</text>
 
+          <text id="mode" fill$="{{ options.modeFill }}" stroke$="{{ options.bezel }}" stroke-width$="{{ options.headingStrokeWidth }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="200px" letter-spacing="0"
+          line-height="125%" word-spacing="0" y="1206" x="460.67117" font-weight="bold" tabindex="0">{{__(mode)}}</text>
```

Comment:
Per other comments, this should be a computed value via a helper function that looks at the isCompass boolean to determine the label text.